great_expectations/expectations/metrics/column_map_metrics/column_values_between.py

```diff
@@ -169,6 +195,29 @@ def is_between(val):
 
         return temp_column.map(is_between)
 
+    @classmethod
+    def _pandas_vectorized(cls, column, min_value, max_value, strict_min, strict_max):
+        if min_value is None:
```

Comment:
👀

Reply:
Looks like that guard is at the very beginning, raising a ValueError on the entire metric.

Reply:
I see. This that is nonlocal to this call site and an assumption it makes, could you add a comment?

Reply:
Happy to do a comment, but on reflection it seems like this is in fact better just checked again, since we've got similar logic again. I'll just add the guard.

Reply:
done.